pysat/_instrument.py

```diff
@@ -1495,9 +1474,6 @@ def load(self, yr=None, doy=None, date=None, fname=None, fid=None,
                 self.meta = self._curr_meta.copy()
             else:
                 self.data = self._null_data.copy()
-                # line below removed as it would delete previous meta, if any
```

Comment:
There must be a better place to keep this knowledge?  Perhaps the docs?

Reply:
I think this would be too detailed for user documentation and only useful for those working on this area of the code

Reply:
I see what you're saying, but disagree.  This is useful to anyone loading multiple days of data into an instrument.  Something that we have a few analysis routines doing, and users would conceivably write code to do.  I think better than keeping these inline comments in a place no-one would think to look is either:
- Fix this by assigning meta towards the end,
- Fix this by assigning meta here if it isn't already assigned,
- Fix this by changing meta so that it isn't overwritten by empty if the last day doesn't have data,
- Fix this by moving it to the docs in a section that talks about loading multiple days.

These solutions could exist in an issue until they're complete.

Reply:
Responding to your comment relevant to this thread that was sent to the general conversation: you not wanting to reintroduce a bug is fine, and could be addressed with a better worded comment (something that would have clued me into why it was important in the first place) that doesn't include commented out code.